ehcache-impl/src/test/java/org/ehcache/config/builders/CacheConfigurationBuilderTest.java

```diff
@@ -285,6 +285,7 @@ public void testOffheapGetsAddedToCacheConfiguration() {
     assertThat(config.getResourcePools().getPoolForResource(ResourceType.Core.OFFHEAP).getUnit(), Matchers.is(MemoryUnit.MB));
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
Personally I would mark tests that are explicitly testing the deprecated functionality as `@Deprecated`, rather than suppressing the warnings.

Reply:
replaced all junit  @SuppressWarnings("deprecation") with @Deprecated, Wil take care of this for upcoming junits